prow/jenkins/jenkins.go

```diff
@@ -396,14 +382,10 @@ func (c *Client) GetBuilds(job string) (map[string]JenkinsBuild, error) {
 	return jenkinsBuilds, nil
 }
 
-// Abort aborts the provided Jenkins build for job. Only running
-// builds are aborted.
+// Abort aborts the provided Jenkins build for job.
 func (c *Client) Abort(job string, build *JenkinsBuild) error {
-	if build.IsEnqueued() {
```

Comment:
Removed since this check will never be true (constrained in the caller). Also, in newer Jenkins versions the following api call works uniformly across enqueued and running builds.